app/views/cart_items/_cart_items.html.erb

```diff
@@ -0,0 +1,26 @@
+<tr>
+
+  <td><%= image_tag item.image_url, height: '300', width: '300', id: "#{item.id}_item" %></td>
+
+  <td><%= item.title %></td>
+
+  <td><%= item.description %></td>
+
+  <td><%= item.travesty_title %></td>
+
+  <td><%= item.price %></td>
```

Comment:
I see that this file was added to solve the issue regarding the local variable 'item' originally being an array. Should we perhaps update the existing partial for that rather than add a new partial which is very similar? Additionally, could use:

<%= number_to_currency(item.price %>

on this line.